src/box/rt_lua_call_access.h

```diff
@@ -0,0 +1,70 @@
+#pragma once
+
+#include <stdint.h>
+#include <stdbool.h>
+#include <zlib.h>
+#include <lua.h>
+#include <lauxlib.h>
+#include <lualib.h>
+
+#if defined(__cplusplus)
+extern "C" {
+#endif /* defined(__cplusplus) */
+
+#define is_super(u) ((u)->roles.m[0] == 20u)
+#define is_universe(s, l) ((l) == 0 || (s)[0] == '\0')
```

Comment:
I would use an in-place `length == 0` check without any extra data lookups.

Reply:
Done.